game/src/info/mod.rs

```diff
@@ -711,32 +711,16 @@ fn make_tabs(
             ctx.style()
                 .btn_tab
                 .text(name)
-                // We use "disabled" to denote "currently selected", but we want to style it like
-                // normal
+                // We abuse "disabled" to denote "currently selected"
                 .disabled(current_tab.variant() == link.variant())
-                .bg_color(ctx.style().btn_tab.bg, ControlState::Disabled)
-                .label_color(ctx.style().btn_tab.fg, ControlState::Disabled)
-                .outline(ctx.style().btn_tab.outline, ControlState::Disabled)
-                // Hide the hit area for selectable tabs unless hovered
-                .bg_color(Color::CLEAR, ControlState::Default)
-                .outline((0.0, Color::CLEAR), ControlState::Default)
-                .bg_color(ctx.style().btn_tab.bg.alpha(0.6), ControlState::Hovered)
                 .build_def(ctx),
         );
         hyperlinks.insert(name.to_string(), link);
     }
     {
-        // stop-gap color that is semi-legible across themes until the tab redesign is completed
```

Comment:
Even though I haven't put the info-tabs into the new tab system, design wise, it's already pretty close. The only visual difference would be adding the "section" background to the tab's content,  instead of a translucent background. 

That might help a little bit, but probably not much. Do you find the contrast in the "data" page to be sufficiently legible?

This is why I offset the panel_bg in https://github.com/a-b-street/abstreet/pull/571/commits/ecca6375a3b357bf1ac70dfc38138e5bd992bdc5 

I could increase that effect to increase contrast, but it's such a key part of the overall design that I'm trying to diverge only as much as necessary to get readable content.

Reply:
> Do you find the contrast in the "more data" page tabs to be sufficiently legible?

Yes, but possibly only because the tabs happen to be on top of a section with the background, so the current tab kind of merges in:
![Screenshot from 2021-03-19 11-44-04](https://user-images.githubusercontent.com/1664407/111828363-6b3d6100-88a8-11eb-8fe7-39ceff6082f0.png)

It still feels hard to distinguish for me, but I'm fine with sticking close to the design for now. Thanks!